actors/power/src/types.rs

```diff
@@ -92,6 +92,20 @@ pub struct MinerRawPowerReturn {
     pub meets_consensus_minimum: bool,
 }
 
+#[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone, Eq, PartialEq)]
+#[serde(transparent)]
+pub struct MinerPowerParams {
+    pub miner: ActorID,
+}
+
+#[derive(Serialize_tuple, Deserialize_tuple, Debug, Clone, Eq, PartialEq)]
+pub struct MinerPowerReturn {
+    #[serde(with = "bigint_ser")]
+    pub quality_adj_power: StoragePower,
+    #[serde(with = "bigint_ser")]
+    pub raw_byte_power: StoragePower,
```

Comment:
This is asymmetric with `MinerRawPowerReturn` but I don't have any suggestions on how to improve it...

Reply:
for one, I think these two should be switched around, typically raw comes before qa so a `[qap,raw]` tuple is atypical

Reply:
The ordering looks good now